### PR TITLE
Switch to concrete classes in experimental asyncio

### LINKED ISSUE #19975: [Aio] Switch to concrete classes
As discussed in gRFC (https://github.com/grpc/proposal/pull/155), we want to migration from using interfaces everywhere to limited interfaces with majority of concrete classes. This issue tracks the execution of that goal.

### PR DESCRIPTION
**Pull request by ZHmao**
Switch to concrete classes, just for experimental asyncio.

### Changes:
Remove interfaces (`Channel ` and `UnaryUnaryMultiCallable `) in `aio/__init__.py` .
Export `Channel` and `UnaryUnaryMultiCallable` from `aio/__init__.py` .

Fixes #19975

### CODE REVIEW

File: src/python/grpcio/grpc/experimental/aio/_channel.py [hunk 1]
```diff
@@ -15,10 +15,10 @@
 
 from grpc import _common
 from grpc._cython import cygrpc
-from grpc.experimental import aio
 
 
-class UnaryUnaryMultiCallable(aio.UnaryUnaryMultiCallable):
+class UnaryUnaryMultiCallable:
+    """Affords invoking a unary-unary RPC from client-side in an asynchronous way."""
```
Comment (lidizheng) [inline src/python/grpcio/grpc/experimental/aio/_channel.py]: s/a/an

Comment (ZHmao) [reply src/python/grpcio/grpc/experimental/aio/_channel.py]: OK

**Comment by pfreixes** [comment]
@lidizheng this would need to be tackled sooner than later https://github.com/grpc/grpc/issues/20117 for making sure that further PR are not breaking the `aio` code, otherwise the unique way of checking this is by running the tests in local.

**Comment by lidizheng** [comment]
@pfreixes I think this change looks good. We need two LGTM from maintainer to merge this PR.
Also, I will raise the priority of making the Kokoro work with Async IO tests.

File: src/python/grpcio/grpc/experimental/aio/_channel.py (hunk 1 above)
Comment (gnossen) [reply src/python/grpcio/grpc/experimental/aio/_channel.py]: Nit: Actually, `s/an/a/`. While, `u` the grapheme is a generally a vowel, the word [is pronounced `/ˈjuːnəɹi/`](https://en.wiktionary.org/wiki/unary), so the first *phoneme*, `j`, is a consonant. Hence "a".

**Comment by lidizheng** [comment]
Rerunning the tests. The EasyCLA bot failed to response in days.

File: src/python/grpcio/grpc/experimental/aio/_channel.py (hunk 1 above)
Comment (ZHmao) [reply src/python/grpcio/grpc/experimental/aio/_channel.py]: Hi Richard, thanks.
`Afford invoking a unary-unary RPC from client-side in an asynchronous way. `
Is it right?

**Comment by lidizheng** [comment]
EasyCLA still stuck, going to close/reopen this PR.

**Comment by lidizheng** [comment]
@thelinuxfoundation CLA bot got stuck for this PR.

**Comment by lidizheng** [comment]
@ZHmao The CLA bot seems decided to misbehave for this pull request. Can you cut a new branch and send another PR? Sorry for the trouble.

**Comment by srini100** [comment]
@ZHmao, did you follow the CLA signing processes? If yes, can you comment or try closing and reopening the PR to trigger CLAbot again?

**Comment by ZHmao** [comment]
Hi @lidizheng @srini100, sorry for delay, I signed CLA just now. I'll try closing and reopening this PR

**Comment by ZHmao** [comment]
I signed it

**Pull request merged**